src/db/tables/onerep_scans.ts

```diff
@@ -28,28 +17,24 @@ export interface GetLatestOnerepScanResult {
   onerep_scan_reason: string;
 }
 
-async function getLatestOnerepScan(
+async function getLatestOnerepScanResults(
   onerepProfileId: number
-): Promise<GetLatestOnerepScanResult | null> {
-  return (
-    await knex("onerep_scans")
-      .select(
-        "onerep_scan_id",
-        "created_at",
-        "updated_at",
-        "onerep_scan_results",
-        "onerep_scan_reason"
-      )
-      .where("onerep_profile_id", onerepProfileId)
-      .orderBy("created_at", "desc")
-      .limit(1)
-  )[0] as unknown as Promise<{
-    onerep_scan_id: number;
-    created_at: number;
-    updated_at: number;
-    onerep_scan_results: { data: ScanResult[] };
-    onerep_scan_reason: "manual" | "initial" | "monitoring";
-  }>;
+): Promise<{ scan: OnerepScanRow | null; results: OnerepScanResultRow[] }> {
+  const scan = await knex("onerep_scans")
+    .first()
+    .where("onerep_profile_id", onerepProfileId)
+    .orderBy("created_at", "desc");
+
+  const results =
+    typeof scan === "undefined"
```

Comment:
nit: is there a diff between `scan ?` vs checking for undefined here?

Reply:
The former does implicit type casting, so `""` would also be falsy, but in practice, there shouldn't be a difference here. I just default to explicit matching on what I expect out of habit, but if there's a reason to prefer the other version, happy to change.